### PR TITLE
Propagate "envFrom" property to CronJob

### PR DESCRIPTION
**Pull request by slamdev**
Existing "secrets" property provides a limited and unsecure way to pass sensitive values to the cron job.

Proposed change allows a user to provide arbitrary secrets\configmaps that are injected as environment variables into the pod.

### CODE REVIEW

File: charts/renovate/templates/cronjob.yaml [hunk 1]
```diff
@@ -59,10 +59,15 @@ spec:
               - name: config-volume
                 mountPath: /usr/src/app/config.json
                 subPath: config.json
-              {{- if .Values.secrets }}
+              {{- if or .Values.secrets .Values.envFrom }}
               envFrom:
+                {{- if or .Values.secrets }}
```
Comment (viceice) [inline charts/renovate/templates/cronjob.yaml:64]: I think the `or` can be removed

Comment (slamdev) [reply charts/renovate/templates/cronjob.yaml]: correct. silly mistake :) fixed

**Comment by viceice** [comment]
Needs deconflicting

**Comment by slamdev** [comment]
> Needs deconflicting

@viceice done

**Review by viceice — APPROVED** [review]
I'm not an helm / kubernetes expert, so requested review from @JamieMagee

**Comment by JamieMagee** [comment]
Yep, this is (one of) the standard ways to do it.

LGTM.

**Comment by slamdev** [comment]
@JamieMagee @viceice not sure guys how the helm release process is configured here and if there is something I missed, but the pipeline is failing in master branch: https://github.com/renovatebot/helm-charts/actions/runs/231353686

**Comment by JamieMagee** [comment]
Yeah, it's expected (for now). Because chart-releaser hasn't detected a version bump, as a change in build (+1) doesn't count, it returns with an error.

Your change will be deployed with the next version bump of the renovate docker image.

**Pull request merged**